packages/react/src/internal/props.ts

```diff
@@ -20,3 +24,8 @@ export type Props<
 export interface EditorProps<T> extends Props<ValueProps<T>, ReadonlyProps, TemplateProps> {
   name?: string
 }
+
+export type FocusableComponent = {
```

Comment:
If onFocus and onBlur is a dom like event can we use typing from DOM?

Reply:
I suggest next typings:

```typescript
export type FocusableComponent = {
  onFocus?: FocusEventHandler<HTMLElement>;
  onBlur?: FocusEventHandler<HTMLDivElement>;
};
```